samples/test-dal-with-nunit/DalWithNunitSample/DAL/PizzaContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace DAL
+{
+    public class PizzaContext : DbContext
+    {
+        private readonly string _connectionString;
+
+        public PizzaContext(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+
+        public DbSet<Pizza> Pizzas { get; set; }
```

Comment:
Move this to use a repository pattern instead.  DbSet<Pizza> should be private to PizzaRepository.